frontend/src/features/public-form/components/FormPaymentPage/stripe/StripeElementWrapper.tsx

```diff
@@ -85,51 +67,69 @@ const StripePaymentContainer = ({
 
   const viewStates = getPaymentViewStates(data?.paymentIntent?.status)
 
-  switch (viewStates) {
-    case PaymentViewStates.Invalid:
-      return (
-        <PaymentBox>
-          <CreatePaymentIntentFailureBlock
-            submissionId={paymentId}
-            paymentClientSecret={paymentInfoData.client_secret}
-            publishableKey={paymentInfoData.publishableKey}
-          />
-        </PaymentBox>
-      )
-    case PaymentViewStates.Canceled:
-      return (
-        <PaymentBox>
-          <span>{viewStates}</span>
-        </PaymentBox>
-      )
-      break
-    case PaymentViewStates.Pending:
-      return (
-        <PaymentBox>
-          <StripePaymentBlock
-            submissionId={paymentId}
-            paymentClientSecret={paymentInfoData.client_secret}
-            publishableKey={paymentInfoData.publishableKey}
-            triggerPaymentStatusRefetch={() => setRefetchKey(Date.now())}
-          />
-        </PaymentBox>
-      )
-    case PaymentViewStates.Succeeded:
-      return (
-        <>
-          <PaymentSuccessSvgr maxW="100%" />
-          <PaymentBox>
+  const renderViewState = () => {
+    switch (viewStates) {
+      case PaymentViewStates.Invalid:
+        return (
+          <PaymentStack>
+            <CreatePaymentIntentFailureBlock
+              submissionId={paymentId}
+              paymentClientSecret={paymentInfoData.client_secret}
+              publishableKey={paymentInfoData.publishableKey}
+            />
+          </PaymentStack>
+        )
+      case PaymentViewStates.Canceled:
+        return (
+          <PaymentStack>
+            <GenericMessageBlock
+              paymentId={paymentId}
+              title={'Payment request was canceled.'}
```

Comment:
```suggestion
              title="Payment request was canceled."
```
nit: since we're just using a string directly, just use quotation marks? There are quite a few instances of this throughout.

Reply:
are you using prettier? eslint should autoformat on save

Reply:
Hm my eslint is happy with it being wrapped with curly braces, does it complain on your side?

Reply:
Actually mine is happy too hahaha

Reply:
Addressed in 87bf06cb